README.md

```diff
@@ -1,56 +1,23 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Porfolio
 
-> Description the project.
+> Portfolio Software Developer.
```

Comment:
[OPTIONAL] It would be better to add a description of your project to your README so your README is documented professionally.

Reply:
thanks @medaminedev66 I’ll keep that in mind for my next delivery